diff -u -r ../mc-4.8.12/src/filemanager/ext.c ./src/filemanager/ext.c
--- ../mc-4.8.12/src/filemanager/ext.c	2014-04-01 12:54:01.000000000 +0200
+++ ./src/filemanager/ext.c	2014-06-05 16:26:24.821998594 +0200
@@ -472,7 +472,7 @@
      * so we clean up after calling view().
      */
     if (!run_view)
-        fprintf (cmd_file, "\n/bin/rm -f %s\n", vfs_path_as_str (script_vpath));
+        fprintf (cmd_file, "\n@/usr/bin/applets/rm -f %s\n", vfs_path_as_str (script_vpath));
 
     fclose (cmd_file);
 
