diff -u -r ../openssh-7.5p1/servconf.c ./servconf.c
--- ../openssh-7.5p1/servconf.c	2017-03-20 03:39:27.000000000 +0100
+++ ./servconf.c	2017-03-24 00:08:05.880913398 +0100
@@ -336,9 +336,13 @@
 
 	assemble_algorithms(options);
 
-	/* Turn privilege separation and sandboxing on by default */
 	if (use_privsep == -1)
+#ifdef __ANDROID__
+		use_privsep = PRIVSEP_OFF;
+#else
+		/* Turn privilege separation and sandboxing on by default */
 		use_privsep = PRIVSEP_ON;
+#endif
 
 #define CLEAR_ON_NONE(v) \
 	do { \
