diff -u -r ../upstream.git/apt-pkg/init.cc ./apt-pkg/init.cc
--- ../upstream.git/apt-pkg/init.cc	2014-05-05 14:01:59.000000000 +0200
+++ ./apt-pkg/init.cc	2014-06-04 13:57:49.848604233 +0200
@@ -45,7 +45,8 @@
       Cnf.Set("APT::Build-Essential::", "build-essential");
    Cnf.CndSet("APT::Install-Recommends", true);
    Cnf.CndSet("APT::Install-Suggests", false);
    Cnf.CndSet("Dir","/");
+   Cnf.CndSet("Acquire::Languages", "none");
    
    // State
    Cnf.CndSet("Dir::State","var/lib/apt/");
