diff -u -r ../mandoc-1.14.3/configure ./configure
--- ../mandoc-1.14.3/configure	2017-08-05 14:40:21.000000000 +0200
+++ ./configure	2017-08-17 16:00:24.385570649 +0200
@@ -40,7 +40,6 @@
 OSNAME=
 UTF8_LOCALE=
 
-CC=`printf "all:\\n\\t@echo \\\$(CC)\\n" | env -i make -sf -`
 CFLAGS="-g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings"
 CFLAGS="${CFLAGS} -Wno-unused-parameter"
 LDADD=
@@ -121,7 +120,7 @@
 INSTALL_LIB=
 INSTALL_MAN=
 INSTALL_DATA=
-LN="ln -f"
+LN="ln -s -f"
 
 # --- manual settings from configure.local -----------------------------
 
@@ -274,8 +273,8 @@
 elif singletest nanosleep NANOSLEEP; then
 	:
 elif runtest nanosleep NANOSLEEP "-lrt"; then
-	LD_NANOSLEEP="-lrt"
+	LD_NANOSLEEP=""
 fi
 if [ "${HAVE_NANOSLEEP}" -eq 0 ]; then
 	echo "FATAL: nanosleep: no" 1>&2
